service/src/intTest/java/uk/nhs/adaptors/gp2gp/common/storageconnectors/StorageConnectorTest.java

```diff
@@ -0,0 +1,56 @@
+package uk.nhs.adaptors.gp2gp.common.storageconnectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class StorageConnectorTest {
+
+    @Autowired
+    private StorageConnector storageConnector;
+
+    private static final String FILE_TO_UPLOAD = "src/intTest/resources/test.txt";
+    private static final String FILE_EXPECTED_DOWNLOAD = "src/intTest/resources/downloads/test.txt";
```

Comment:
@mmackiewicz1 change file name to reflect contents

Reply:
Discussed with Thomas, leaving for further stories